### DISCUSSION TITLE
How to add command line arguments to One Click Installer version

### DISCUSSION THREAD

**Discussion by eo407** [Sanster/IOPaint#159] [Q&A]
Hi, I am not super savvy so I am using the One Click Installer mentioned here:
https://github.com/Sanster/lama-cleaner/blob/main/scripts/README.md

I would like to be be able to use the arguments like xformers in that version but I cannot figure out where I would do that. Do I add it to win_start.bat in a particular syntax? Thanks!

**Comment by Sanster** [comment]
You can add extra arguments at [tasks.py](https://github.com/Sanster/lama-cleaner/blob/main/scripts/user_scripts/tasks.py#L124)

**Reply by Rogal80** [reply]
Hi, but where in task.py we need to put arguments?

**Reply by virgilianshailer** [reply]
did you find answer?

**Reply by Sanster** [reply]
Sorry for the late reply, you can add command line arguments in the `c.run()`, e.g `lama-cleaner --model {model} --device {device} --port {port} --gui --gui-size 1400 900 --sd-cpu-textencoder`

<img width="819" alt="image" src="https://user-images.githubusercontent.com/3998421/211700301-323ef23a-a31a-4ae8-97f8-975806717cca.png">
